src/core/sprites/webgl/generateMultiTextureShader.js

```diff
@@ -35,6 +36,7 @@ export default function generateMultiTextureShader(gl, maxTextures)
 
     shader.bind();
     shader.uniforms.uSamplers = sampleValues;
+    shader.uniforms.uLODBiases = Array(maxTextures).fill(0);
```

Comment:
Don't think we can use this function as it isn't supported by all our browsers. Probably easier to create a `new Float32Array(maxTextures)` which will default to zeroes.

Reply:
Yes, that would be better.